src/oled_display_node.cpp

```diff
@@ -190,118 +204,122 @@ static uint8_t sh1106_init_bytes[SH1106_INIT_BYTE_COUNT] = {
 };
 
 /*
- * @name                dispOled_writeBytes
- *
- * @name                display_setCursor
- * @brief               Move cursor to a given horizontal column and line
- *
- * @param               dispCtx         Context for display that holds type and hardware interface info
- * @param               column          The pixel resolution column from 0 to 127
- * @param               line            The line number where 0 is top line
+ * @name                i2c_read
```

Comment:
Your swap of i2c read across from dispOled_write makes the review less effective because they are not even the same functionalities but I understand and there is no need to change that.  It just makes the review compare of files close to pointless so I simply look at new code.
DO NOT REFACTOR IT!   I TESTED IT IN THIS FORM SO KEEP IT LIKE IT IS NOW.  Just a comment that it made the review more time consuming.